packages/hardhat-core/test/fixture-projects/solidity-config-warnings/remapping-in-list.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  solidity: {
+    compilers: [
+      {
+        version: "0.7.0",
+        settings: {
+          remappings: [":g=/dir"],
+        },
+      },
+    ],
+    overrides: {
```

Comment:
oh, i almost forgot this, but the overrides can also have remappings

Reply:
ahh good catch